website/docs/reference/resource-properties/arguments.md

```diff
@@ -0,0 +1,117 @@
+---
+title: arguments
+sidebar_label: "arguments"
+id: arguments
+pagination_next: "reference/resource-properties/arguments"
+pagination_prev: null
+---
+
+<File name='macros/<filename>.yml'>
+
+```yml
+
+version: 2
+
+macros:
+  - name: <macro name>
+    arguments:
+      - name: <arg name>
+        type: <string>
+        description: <markdown_string>
+
+```
+
+</File>
+
+## Definition
+
+The `arguments` property is used to define the parameters that a macro can accept. Each argument can have a `name`, `type`, and `description`. This helps in documenting the macro and understanding what inputs it requires.
+
+You can use a [behavior flag](/reference/global-configs/behavior-changes#behavior-change-flags) in v1.10; dbt will validate whether the `name` or `type` matches the marcro arguments in YAML.
+
+`arguments` can be defined in a [`properties.yml`](/reference/resource-properties/arguments#macro-properties) file but they are not enforced by dbt. The file can be named anything and placed in the [`macro-paths`](/reference/project-configs/macro-paths) directory (defaults to `["macros"]`).
```

Comment:
1. Can you make these sentences active? For example, "You can define `arguments`..." and " You can name this file..."
2. I wonder if this paragraph should move up to before the behavior flag paragraph? It might flow better.